project/npda/templates/partials/view_preference.html

```diff
@@ -27,7 +27,7 @@
             hx-target="#global_view_preference"
             hx-trigger="change"
             hx-headers='{"X-CSRFToken": "{{ csrf_token }}"}'
-            hx-include="[name='view_preference'], [name='pz_code_select_name']">
```

Comment:
I'm not sure how this was working before but I was getting a bug where it was pulling up the `view_preference` back to 2 when changing the PDU